cmd/utils/flags.go

```diff
@@ -205,7 +205,7 @@ var (
 	defaultSyncMode = ethconfig.Defaults.SyncMode
 	SyncModeFlag    = TextMarshalerFlag{
 		Name:  "syncmode",
-		Usage: `Blockchain sync mode ("fast", "full", "snap" or "light")`,
+		Usage: `Blockchain sync mode ("snap", "full" or "light")`,
```

Comment:
Might be that this change causes a lot of existing setups to refuse to start. Maybe we shoud deprecate `fast` and just internally translat it to `snap` for a while?